### DISCUSSION TITLE
屏蔽登录窗口

### DISCUSSION THREAD

**Discussion by hupo-jun** [the1812/Bilibili-Evolved#3272] [功能建议 / Ideas]
如题，打开b站，每次都会提示登录，特别烦人。不管是首页还是视频页面，只要网址变了，都有这个登录提醒，希望角本能把位于网页右上角和右下角的登录屏蔽调
![Screenshot from 2022-04-23 20-07-50](https://user-images.githubusercontent.com/92795607/164893931-902cfc58-d604-420b-88e6-2468bfe0e02e.png)
![Screenshot from 2022-04-23 20-07-13](https://user-images.githubusercontent.com/92795607/164893945-e91729dc-68a2-4a48-b38b-ddaa71237c80.png)

**Comment by theblueandblackcrow** [comment]
所以你为什么不登录来消去这个烦人的弹窗呢？

**Comment by ShuiHuo** [comment]
现在不登录不止有弹窗，还会暂停视频。希望能出个屏蔽登录窗口的功能，毕竟我没B站账号，也不想登录看视频。
